client/boot/index.js

```diff
@@ -1,5 +1,9 @@
 // Initialize localStorage polyfill before any dependencies are loaded
 require( 'lib/local-storage' )();
+
+// Initialize draft-js polyfill needed for SEO title format editor
+require( 'lib/draft-js-polyfill' )();
```

Comment:
To expand a bit, is the reason for the issue here that draft-js is being loaded in a chunk different than the one SEO lives at? If so, could we look at making it part of the same bundle and load the polyfill only at the start of that chunk?

The main issue is that loading at the start of boot doesn't take into account that most sections of Calypso don't need to run draft-js at the moment.